tests/vm_test.go

```diff
@@ -1016,12 +1016,12 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			It("Should force stop a VMI", func() {
 
 				By("getting a VM with high TerminationGracePeriod")
-				newVMI := tests.NewRandomVMI()
-				gracePeriod := int64(1600)
-				newVMI.Spec.TerminationGracePeriodSeconds = &gracePeriod
-
+				newVMI := libvmi.New(
+					libvmi.WithResourceMemory("32Mi"),
```

Comment:
Based on what you chose 32M? Why not 1M? 2M?

Reply:
The decision to use 32Mi came from this comment:
https://github.com/kubevirt/kubevirt/pull/8075#discussion_r919005138